### DISCUSSION TITLE
Mill "snapshot" releases... ?

### DISCUSSION THREAD

**Discussion by Quafadas** [com-lihaoyi/mill#3254] [General]
I'm not pretending to understand exactly what's happening here, but mill has gotten very active in my GitHub feed. 

![image](https://github.com/com-lihaoyi/mill/assets/24899792/7d20f965-d037-4850-8a3b-5f32eb52dc65)

It's not a "problem" per se - I usually get quite excited when I see a mill release - but I don't think I should be excited about these ones? I guess I'm wondering if it is intentional that they get advertised rather prominently... 

It isn't something I'm worried about, so it may well be "intentional - nothing to see here"...  I'm just checking :-).

**Comment by lihaoyi** [comment]
Just trying to debug some issues internally, not intended for wider use

**Comment by lefou** [comment]
I think the issue was, that these releases were marked as "latest release". I update the `0.11.8` to fix that.
